_posts/blog/2019-08-12-why-upgrade.md

```diff
@@ -69,12 +73,19 @@ try (AccumuloClient client = Accumulo.newClient()
 ```
 
 As you can see the client is also closable, which gives developers more control over resources.
-See the [Accumulo entry point javadoc][client] for more info. 
+See the [Accumulo entry point javadoc][client].
+
+Key and Mutation have new fluent APIs:
```

Comment:
Could explicitly mention that these new APIs make it easy to mix types like `String` and `byte[]`.  This was cumbersome in previous APIs